plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/env/ChooseProcessTypeDialog.java

```diff
@@ -132,6 +128,9 @@ public void widgetDefaultSelected(SelectionEvent e) {
             }
         });
 
+        checkboxSaveSelection = new Button(area, SWT.CHECK);
+        checkboxSaveSelection.setToolTipText("Saves the selection from this dialog.");
```

Comment:
The tooltip can say:

`"When saving the selection from this dialog in the preferences, the selected action will be automatically applied. Later on it's possible to change the setting at the preferences under PyDev > Interactive Console"`.